weave_demo.ipynb

```diff
@@ -0,0 +1,1199 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "286b1cb7-6abc-4707-bc30-a163aec301a2",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['/home/jovyan/work/weave/weave-demo-pantry/test-1/acc530466eb811eeb8b60242ac1f000a/WeaveDemoText.txt']"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "basket_contents = basket.ls()\n",
+    "basket_contents"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ebde9549-4646-4964-87d7-389b874f8982",
+   "metadata": {},
+   "source": [
+    "## Data Provenance\n",
+    "\n",
+    "Weave handles much of its data provenance tracking through the creation of baskets. A basket is meant to represent an atomic data product. It can contain whatever a user wishes to put in the basket, but it's intended purpose is to hold a single instance of one type of data, be it an image, video, text file, or curated training set. A basket in its entirety contains the actual data files specified by the user along with the supplemental files that Weave creates. These supplemental files contain data integrity information, arbitrary metadata specified by the user, and lineage artifacts. Baskets are created at their time of upload and uploaded in an organized state to the data store. Parent UUIDS are the UUIDS Basket(s) that created the current basket."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "fed30aa8-3224-4cdb-b708-047dd986cbda",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "Empty DataFrame\n",
+       "Columns: []\n",
+       "Index: []"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "pantry1.index.get_parents(pantry1_df['uuid'][0])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "57096bff-4f16-4718-a078-20cd196e56ab",
+   "metadata": {},
+   "source": [
+    "There are currently no parents or children associated with our pantry, just as we expected. Let's create some parents/children and then check this functionality out again."
```

Comment:
Call out the difference here between the first upload. "Notice the new basket is a new type and we supply a parent UUID to indicate it is derived from the previous basket".... not exactly that, but that idea.